### PR TITLE
fix dependency scopes

### PR DESCRIPTION
**Pull request by patriknw**
* included dependency to core-tests and also a testkit dependency as full dependencies

References #409

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -55,8 +55,8 @@ lazy val cassandra =
     .settings(Defaults.itSettings)
     .settings(Dependencies.cassandra)
     .dependsOn(core)
-    .dependsOn(coreTest % "test->test;compile->test;")
-    .dependsOn(testkit % "compile->compile;test->test")
+    .dependsOn(coreTest % "test->compile;it->test")
+    .dependsOn(testkit % "test->compile;it->compile")
```
Comment (renatocaval) [inline build.sbt:59]: Shouldn't it be `test->test`? 

As I understand it, it is saying that the cassandra module depends on the test code in `coreTest` and `testKit`, but it should depend only on test, no? Or am I flipping things around?

Comment (renatocaval) [reply build.sbt:59]: Or maybe it should be `compile->test` because we are want the code from coreTest and testkit available in test scope

Comment (patriknw) [reply build.sbt:59]: I think I tried various combination, but maybe I got it wrong. The pom.xml is correct with this anyway.

**Review by renatocaval — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (renatocaval) [reply build.sbt:59]: Ok, let's go for it. 

I restart the build. Failed because of some flakiness.

**Pull request merged**